build.gradle.kts

```diff
@@ -2,11 +2,11 @@ import org.jetbrains.kotlin.gradle.plugin.getKotlinPluginVersion
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    kotlin("jvm") version "1.6.20"
-    id("com.gradle.plugin-publish") version "0.18.0"
+    kotlin("jvm") version "1.7.0"
+    id("com.gradle.plugin-publish") version "0.21.0"
     `java-gradle-plugin`
     `maven-publish`
-    id("org.jmailen.kotlinter") version "3.10.0"
+    id("org.jmailen.kotlinter") version "3.11.1"
```

Comment:
Should we bundle this with the next ktlint release?

Reply:
I might be missing something, but that's a plugin's build dependency. It doesn't affect the produced artifact in any way. Running `./gradlew :dependencies --no-configuration-cache | grep kotlinter` returns no kotlinter dependency present.